Javadoc is failing in Travis CI for java 11

https://travis-ci.org/dapengzhang0/grpc-java/jobs/480570158
```
> Task :grpc-auth:javadoc
javadoc: warning - You have specified the HTML version as HTML 4.01 by using the -html4 option.
The default is currently HTML5 and the support for HTML 4.01 will be removed
in a future release. To suppress this warning, please ensure that any HTML constructs
in your comments are valid in HTML5, and remove the -html4 option.
javadoc: error - The code being documented uses modules but the packages defined in https://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.
1 error
1 warning
> Task :grpc-auth:javadoc FAILED
> Task :grpc-compiler:linkJava_pluginExecutable
FAILURE: Build failed with an exception.
```

This might be related to https://bugs.openjdk.java.net/browse/JDK-8212233

--- a/build.gradle
+++ b/build.gradle
@@ -98,6 +98,7 @@ subprojects {
         encoding = 'UTF-8'
         use = true
         links 'https://docs.oracle.com/javase/8/docs/api/'
+        source = "8"
     }
 
     ext {

